spec/models/timeline_spec.rb

```diff
@@ -163,6 +163,9 @@
       it 'does not return timelines without matching titles' do
         expect(Timeline.title_like(title_filter)).not_to include(timeline3)
       end
+      it 'searches on multiple terms' do
+        expect(Timeline.title_like('Fav Moose')).to include(timeline2)
+      end
```

Comment:
Not a blocker but, since I think the expected behavior would be that it only returns timeline 2, should we have negative tests as well to make sure that timeline 1 and 3 are not included in `title_like`? Or do I have the expected behavior wrong? 

Also, we can maybe see what the behavior is in testing, but I think these changes may not care about word order? If that's the case, and it is the behavior we want to maintain in the future, it may be good to add a check that something like "Moose Fav" also returns timeline 2.